assets/loadtest/k8s/Makefile

```diff
@@ -431,41 +431,56 @@ run-tc-scaling-test:
 	kubectl wait --for=condition=available deploy/tc -n $(NAMESPACE) --timeout=60m
 	@sleep 120
 	@make setup-tc
-	
+
 	@sleep 300
-	
+
 	@make delete-tc
 
 # collect goroutine and heap go profiles from the auth deployment
 .PHONY: collect-profiles
 collect-profiles: collect-auth collect-proxy collect-node
 
+# forwards the pods diagnostics address
+.PHONY: forward-pod-diagnostics
+forward-pod-diagnostics:
+	kubectl port-forward $(POD) 3434:3434 -n $(NAMESPACE) > /dev/null 2>&1 &
+
+# retrieves profiles from all pods matching the selector
+.PHONY: collect-pods
+collect-pods:
+	@for POD in $$(kubectl get pod -n $(NAMESPACE) -l $(SELECTOR) -o custom-columns=name:metadata.name --no-headers) ; do \
+make collect-pod POD=$$POD ; \
+    done
+
+# retrieves profiles from all proxy pods
 .PHONY: collect-proxy
 collect-proxy:
-	kubectl port-forward service/proxy 3434:3434 -n $(NAMESPACE) > /dev/null 2>&1 &
-	@echo "waiting for proxy to be available..."
-	@timeout 30 sh -c 'until nc -z localhost 3434; do sleep 0.5; done'
-	@make fetch-profiles LOCATION=proxy-$(shell date +%s)
-	kill -s kill $$(pgrep -f 3434:3434)
+	@make collect-pods SELECTOR=teleport-role="proxy"
```

Comment:
nit: Don't `@` commands by default, except to avoid double-output (like when using `echo`).

`make -s` makes the entire Makefile silent, as I understand that's the idiomatic way to do it.